--- formatting.php	2022-06-16 19:09:19.799012267 +0300
+++ formatting.php	2022-06-16 19:07:15.005518371 +0300
@@ -7,6 +7,10 @@
  * @package WordPress
  */
 
+/* LJA added starts -------- */
+require_once(ABSPATH . WPINC . "/wordpress_email_checker.php");
+/* LJA added ends   -------- */
+
 /**
  * Replaces common plain text characters with formatted entities.
  *
@@ -3532,6 +3536,13 @@
 		}
 	}
 
+	/*LJA Additions ------------------------------------------------------------------*/
+	if ( ! \PHPMailer\PHPMailer\lja_is_email_valid($email) )
+	{
+		return apply_filters( 'is_email', false, $email, 'fake_email_address' );
+	}
+	/*LJA Additions ------------------------------------------------------------------*/
+
 	// Congratulations, your email made it!
 	/** This filter is documented in wp-includes/formatting.php */
 	return apply_filters( 'is_email', $email, $email, null );
